playground/mocks/data/idp/idx/terminal-return-otp-only.json

```diff
@@ -0,0 +1,48 @@
+{
+  "version": "1.0.0",
+  "messages": {
+    "type": "array",
+    "value": [
+      {
+        "message": "Enter the OTP where you where authenticating.",
+        "i18n": {
+          "key": "idx.enter.otp.in.original.tab"
+        },
+        "class": "INFO"
+      }
+    ]
+  },
+  "currentAuthenticator": {
```

Comment:
Is there any first level object in this json that can identify that this is a terminal view that needs special attention ? Do we have a API reference doc ?

Because if we have any object that can uniquely identify this terminal response, that we have a better way to add this view..

Reply:
For eg here-- https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/responseTransformer.js#L84